packages/now-routing-utils/src/superstatic.ts

```diff
@@ -181,7 +183,24 @@ function replaceSegments(
       indexes[name] = toSegmentDest(index);
     });
 
+    let destParams: string[] = [];
+
     if (destination.includes(':') && segments.length > 0) {
+      const pathnameKeys: Key[] = [];
+      const hashKeys: Key[] = [];
+
+      try {
+        pathToRegexp(pathname, pathnameKeys);
+        pathToRegexp(hash || '', hashKeys);
+      } catch (_) {
+        // this is not fatal so don't error when failing to parse the
+        // params from the destination
+      }
+
+      destParams = [...pathnameKeys, ...hashKeys]
+        .map(key => key.name)
+        .filter(isString);
```

Comment:
```suggestion
      destParams = new Sett([...pathnameKeys, ...hashKeys]
        .map(key => key.name)
        .filter(isString));
```

Reply:
Probably wont see much of a difference here but maybe depending on how many keys are used

Reply:
Tested no Node 12 and there is no difference between Array.includes and Set.has when the items are less than 5 🤷‍♂️